content/applications/finance/accounting/bank/setup/bank_accounts.rst

```diff
@@ -1,95 +1,81 @@
+:menuselection:`Configuration --> journals`. Open it to edit the different information related to
+your bank account.
 
-- **Name**: the bank account's name, as displayed on Odoo.
-- **Account Number**: your bank account number (IBAN in Europe).
-- **Bank**: click on *Create and Edit* to configure the bank's details. Add the bank institution's
-  name and its Identifier Code (BIC or SWIFT).
-- **Code**: this code is your Journal's *Short Code*, as displayed on Odoo. By default, Odoo creates
-  a new Journal with this Short Code.
-- **Journal**: This field is displayed if you have an existing Bank Journal that is not linked yet
-  to a bank account. If so, then select the *Journal* you want to use to record the financial
-  transactions linked to this bank account or create a new one by clicking on *Create and Edit*.
+.. image:: bank_accounts/bank_account_manual_config.png
+   :align: center
+   :alt: Manually configure your bank information
+
+To manually **add** another bank account, go to :menuselection:`Accounting --> Configuration --> Add
+a Bank Account`, and click on :guilabel:`Create it` (at the bottom right). Fill out the form.
 
 .. note::
-   Odoo detects the bank account type (e.g., IBAN) and enables some features accordingly.
+   Odoo detects your bank account type (e.g., IBAN) and enables some features accordingly.
 
-.. image:: bank_accounts/bank_accounts_manual.png
-   :align: center
-   :alt: Add manually a new bank account in Odoo Accounting
 .. _bank_accounts_configuration:
 
 Advanced configuration
 ======================
 
-To edit an existing bank account, go to :menuselection:`Accounting --> Configuration --> Bank
-Accounts`, and open the bank account you want to modify.
+To **edit an existing bank account** go to :menuselection:`Accounting --> Configuration --> Journals`
+and select the bank journal you want to modify.
 
-If you need to edit the bank account details, go to the *Bank Account* field and click on the
-*External Link* button next to the list arrow. There, you can edit the bank account's number,
-Account Holder, Account Holder Name, and your Bank Institution's details by clicking on the
-*External Link* next to the *Bank* field. These details are used to register some payments.
+If you need to **edit your bank account details**, click on the :guilabel:`Account number`. A new
+page opens. Click on the external sign next to the :guilabel:`Bank` field and update your bank
+information accordingly. These details are used when registering payments.
 
-You can configure which types of payments are enabled in the **Payment Method Types** section and
-how the bank statements are recorded and posted in the **Bank Statements** section.
-
-.. image:: bank_accounts/bank_accounts_configuration.png
-   :align: center
-   :alt: Advanced configuration of a bank account in Odoo Accounting
+The :guilabel:`Bank Feeds` can be selected from the bank journal. 3 options are possibles:
```

Comment:
```suggestion
:guilabel:`Bank Feeds` can be selected from the bank journal. Three options are available:
```
Numbers from zero to nine are usually spelled out. Always spell out numbers when they start a sentence.